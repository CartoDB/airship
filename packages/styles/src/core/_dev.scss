@import 'tools/_mixins';
@import 'variables/_variables';
