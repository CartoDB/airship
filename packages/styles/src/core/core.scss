@import 'variables/_variables';

* {
  box-sizing: border-box;
  font-family: sans-serif;
}

:root {
  --primary-color: #{$primary-color};
}
