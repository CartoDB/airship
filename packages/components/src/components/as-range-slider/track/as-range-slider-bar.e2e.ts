import { E2EPage, newE2EPage } from '@stencil/core/testing';

describe('as-range-slider-bar', () => {
  let page: E2EPage;

  beforeEach(async () => {
    page = await newE2EPage();

    await page.setContent(`
      <as-range-slider-bar></as-range-slider-bar>
    `);
  });

  describe('Rendering', () => {
    it('should render a bar with its values', async () => {
      const element = await page.find('as-range-slider-bar');
      element.setProperty('rangeStartPercentage', 20);
      element.setProperty('rangeEndPercentage', 80);
      element.setProperty('stepPercentage', 10);
      await page.waitForChanges();

      expect(element.outerHTML).toMatchSnapshot();
    });
  });
});
