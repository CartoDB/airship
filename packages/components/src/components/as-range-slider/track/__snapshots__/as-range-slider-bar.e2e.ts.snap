// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as-range-slider-bar Rendering should render a bar with its values 1`] = `"<as-range-slider-bar class=\\"hydrated\\"><div class=\\"as-range-slider__range-bar\\" style=\\"left: 20%; width: 60%;\\"></div></as-range-slider-bar>"`;
